docs/var.tfvars-doc.md

```diff
@@ -211,6 +211,7 @@ Change the values as per your requirement.
 ```
 fips_compliant      = false
 ```
+Note: Once fips_compliant set to true it will enable OCP cluster to FIPS enable but bastion node also should be FIPS enabled for that required code changes are added in prepare module and for replicating those changes in bastion node it require to reboot bastion node, that code changes are added at the end in install module.
```

Comment:
Could be simpler. 
```
Note: Once fips_compliant set to true it will enable FIPS on the OCP cluster and also on bastion nodes. At the end of install the bastion nodes will be rebooted.
```

Reply:
@yussufsh  Done